r/tests/testthat/test-dplyr.R

```diff
@@ -815,7 +815,18 @@ test_that("type checks on expressions", {
         a = is.character(as.character(int)),
         b = is.integer(as.character(int)),
         c = is.integer(int + int),
-        d = is.double(int + dbl),
+        d = is.double(int + dbl)
```

Comment:
If you wanted to confirm that `is.logical` works you could do:

```
e = is.logical(chr %in% c("d", "e", "f"))
```

Reply:
Sounds good. I'll keep it simpler and use a numeric inequality instead.